static/app/views/issueDetails/groupSimilarIssues/index.spec.tsx

```diff
@@ -30,7 +31,7 @@ describe('Issues Similar View', function () {
     {
       router: {
         ...RouterFixture(),
-        params: {orgId: 'org-slug', projectId: 'project-slug', groupId: 'group-id'},
+        params: {orgId: 'org-slug', projectId: 'project-slug', groupId: '1000000'},
```

Comment:
chose an arbitrarily large number in hopes that it is unique. I need it to be a number for the analytics